server/service-unavailable-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "start": "2021-06-26T07:00:00+01:00",
+  "end": "2021-06-26T21:00:00+01:00",
+  "message": "Some parts of Refer and monitor an intervention are unavailable until 9:00pm. You can read referral and intervention information but any updates you make will not be recorded."
```

Comment:
_not a blocker_ 🙋‍♂️ we could pull this directly from somewhere at startup (to make it changeable without deploy)

Reply:
should we have a probationMessage and a providerMessage?

Reply:
It might be useful in future, but for its current purpose we'll be displaying the same message to both sets of users. Shall we see if we need it further down the line?